src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Utils/__tests__/DCCDateStringFormatterTests.swift

```diff
@@ -8,53 +8,67 @@ import XCTest
 class DCCDateStringFormatterTests: XCTestCase {
 
 	func testGIVEN_UnformattedDateOfBirth_When_Formatting_THEN_FormatIsCorrect() throws {
-		for dateOfBirthFormattingTestData in dateOfBirthFormattingTestDatas {
+		for dateOfBirthFormattingTestData in dateOfBirthFormattingTestData {
 			guard let dob = dateOfBirthFormattingTestData["dob"] else {
 				XCTFail("String expected.")
 				return
 			}
-			let formattedDOB = DCCDateStringFormatter.formatedString(from: dob)
+
+			let formattedDOB = DCCDateStringFormatter.formattedString(from: dob)
 			let expected = dateOfBirthFormattingTestData["formatted"]
 			XCTAssertEqual(formattedDOB, expected)
+
+			let localizedFormattedDOB = DCCDateStringFormatter.localizedFormattedString(from: dob)
+			let expectedLocalizedFormat = dateOfBirthFormattingTestData["localized"]
+			XCTAssertEqual(localizedFormattedDOB, expectedLocalizedFormat)
 		}
 	}
 
-	let dateOfBirthFormattingTestDatas = [
```

Comment:
Just a minor thing. I know its not correct from the grammar perspective. But it was intended to communicate that its an array. So either we use "dateOfBirthFormattingTestDatas" or we change the name to something where plural is distinguishable. Maybe  "dateOfBirthFormattingTestEntries"